### PR TITLE
ivtest: Document the ivest/regress-vNN.list file.

### PR DESCRIPTION
**Pull request by steveicarus**
This is another bit that needs to be accounted for when changing versions.

### CODE REVIEW

File: Documentation/developer/version_stamps.rst [hunk 1]
```diff
@@ -12,18 +12,28 @@ When versions are changed, the above files need to be edited to account for
...
 
-This now includes version_base.h to get the version
+This now includes version_base.h to get the version:
 
 * vpi/vams_simparam.c -- Hard coded result to simulatorVersion query
+
+This is actually a test file list that is specific to a major version.
+The regression test scripts query the version of the compiler to infer
+that it must include this list of tests. For example, for version 12.x
+of the compiler, the needs to be an ivltets/revress-v12.list file that
```
Comment (larsclausen) [inline Documentation/developer/version_stamps.rst:36]:
```suggestion
of the compiler, the needs to be an ilvtests/regress-v12.list file that
```

File: Documentation/developer/version_stamps.rst [hunk 2]
```diff
@@ -12,18 +12,28 @@ When versions are changed, the above files need to be edited to account for
 the new version information. The following used to have verion information in
 them, but now their version information is generated:
 
-Replaced with version_base.h, which is edited manually, and
-version_tag.h which is generated from git tag information.
+The version_tag.h file is generated from git tag information using
+the "make version" target, or automatically if the version_tag.h
+file doesn't exist at all. This implies that a "make version" is
+someting worth doing  when you do a "git pull" or create commits.
```
Comment (larsclausen) [inline Documentation/developer/version_stamps.rst:18]:
```suggestion
something worth doing when you do a "git pull" or create commits.
```

File: Documentation/developer/version_stamps.rst (hunk 1 above)
Comment (steveicarus) [reply Documentation/developer/version_stamps.rst]: Fixed.

File: Documentation/developer/version_stamps.rst (hunk 2 above)
Comment (steveicarus) [reply Documentation/developer/version_stamps.rst]: Fixed.

**Pull request merged**